src/components/select/select-list/select-list.spec.js

```diff
@@ -0,0 +1,246 @@
+          act(() => {
+            domNode.dispatchEvent(downKeyDownEvent);
+          });
+          act(() => {
+            domNode.dispatchEvent(downKeyDownEvent);
+          });
+          expect(wrapper.update().find(Option).at(1)).toHaveStyleRule('background-color', baseTheme.select.selected);
+        });
+      });
+
+      describe('with the last option already highlighted', () => {
+        it('then the first Option should be highlighted', () => {
+          wrapper.setProps({ highlightedValue: 'opt3' });
+
+          act(() => {
+            domNode.dispatchEvent(downKeyDownEvent);
+          });
+
+          expect(wrapper.update().find(Option).first()).toHaveStyleRule('background-color', baseTheme.select.selected);
+        });
+      });
+    });
+
+    describe("and it's the Up key", () => {
+      describe('with no option highlighted', () => {
+        it('then the last Option should be highlighted', () => {
+          act(() => {
+            domNode.dispatchEvent(upKeyDownEvent);
+          });
+          expect(wrapper.update().find(Option).last()).toHaveStyleRule('background-color', baseTheme.select.selected);
+        });
+      });
+
+      describe('double pressed', () => {
+        it('then the second Option should be highlighted', () => {
+          act(() => {
+            domNode.dispatchEvent(upKeyDownEvent);
+          });
+          act(() => {
+            domNode.dispatchEvent(upKeyDownEvent);
+          });
+          expect(wrapper.update().find(Option).at(1)).toHaveStyleRule('background-color', baseTheme.select.selected);
+        });
+      });
+
+      describe('with the first option already highlighted', () => {
+        it('then the last Option should be highlighted', () => {
+          wrapper.setProps({ highlightedValue: 'opt1' });
+
+          act(() => {
+            domNode.dispatchEvent(upKeyDownEvent);
+          });
+
+          expect(wrapper.update().find(Option).last()).toHaveStyleRule('background-color', baseTheme.select.selected);
+        });
+      });
+    });
+
+    describe("and it's the Home key", () => {
+      it('then the first Option should be highlighted', () => {
+        act(() => {
+          domNode.dispatchEvent(homeKeyDownEvent);
+        });
+        expect(wrapper.update().find(Option).first()).toHaveStyleRule('background-color', baseTheme.select.selected);
+      });
+    });
+
+    describe("and it's the End key", () => {
+      it('then the last Option should be highlighted', () => {
+        act(() => {
+          domNode.dispatchEvent(endKeyDownEvent);
+        });
+        expect(wrapper.update().find(Option).last()).toHaveStyleRule('background-color', baseTheme.select.selected);
+      });
+    });
+
+    // coverage filler
+    it('does nothing if the other key is pressed', () => {
+      act(() => {
+        domNode.dispatchEvent(new KeyboardEvent('keydown', { which: 32, bubbles: true }));
+      });
```

Comment:
I realise this test is merely to hit coverage but maybe you could iterate over the `Options` and `expect(option).not.toHaveStyleRule('background-color')`, then at least that way there's is an assertion at the end of it

Reply:
added new assertion